aarch64/src/l.S

```diff
@@ -0,0 +1,29 @@
+// Aarch64 entry (Raspberry Pi 3, 4 focussed)
+ 
+.section .boottext, "awx"
+.globl start
+
+start:
+	// All cores other than 0 should just hang
+    mrs     x1, mpidr_el1
+    and     x1, x1, #3
+    cbnz    x1, 5f
+	
+	// On the main core, so set up the initial stack
+	ldr	x5, =start
```

Comment:
This won't really work once we start using virtual memory. Presumably, this code will end up in the text segment and be mapped R/O. We should probably reserve some space for an initial stack in low RAM like we do for x86.

Reply:
Yeah, this code has been written without thinking hard about virtual memory - just wanted to get something in that worked, so that we're thinking cross-platform from the very start.  Now that we have something that works in this basic state, we can improve.